### PR TITLE
Add progress bar percentage display

### PR DESCRIPTION
**Pull request by 3kh0**
This pull adds a few new words and phrases, improves the README a bit, and adds a percentage to the progress bar

### CODE REVIEW

**Comment by maxwofford** [comment]
Please run the formatter provided with `bun fmt`! I think your editor is autoformatting the files you opened and changing stuff you don't mean to. Heads up, you probably never want to `git add --all` or commit whole file changes– check out `git add -p`.

**Comment by 3kh0** [comment]
@maxwofford Fixed!

File: README.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ High Seas is the successor to [Arcade](https://hackclub.com/arcade/), Hack Club
 
 ## Local development
 
-Set sail with a simple `bun i; bun run dev`
+Set sail with these commands:
+
+```sh
+bun i
+bun run dev
```
Comment (malted) [inline README.md:19]: Let's keep this as it was

File: lib/flavor.js [hunk 2]
```diff
@@ -32,6 +33,8 @@ export const purchaseWords = [
   'Plunder',
   'Plunder',
   'BitTorrent',
+  'Pirate',
+  'Pirate',
```
Comment (malted) [inline lib/flavor.js:37]:
```suggestion
```

File: lib/flavor.js [hunk 3]
```diff
@@ -43,6 +46,17 @@ export const cantAffordWords = [
...
+  'not in your budget...',
+  "you can't afford that...",
+  'too expensive...',
+  // "can't afford this...",
+  "can't afford...",
+  'unaffordable...',
+  'out of reach...',
+  'need doubloons...',
+  'too pricey...',
+  // "when you're richer...",
```
Comment (malted) [inline lib/flavor.js:59]: Have you tested this? We shortened these pretty aggressively because the buttons are so small

Comment (malted) [inline lib/flavor.js:59]: Repeated

```suggestion
```

Comment (malted) [reply lib/flavor.js:59]: image

Case in point

File: src/app/harbor/shop/progress.tsx [hunk 4]
```diff
@@ -19,19 +19,25 @@ export default function Progress({ val, items }) {
...
-                  : '100') + '%',
+              width: progressPercentage + '%',
             }}
           ></div>
+          <div
+            className="absolute top-0 left-1/2 transform -translate-x-1/2 text-white font-bold"
+            style={{ zIndex: 1 }}
+          >
+            {Math.round(progressPercentage)}%
+          </div>
```
Comment (malted) [inline src/app/harbor/shop/progress.tsx:40]: image

The reason this wasn't there before is that it's not just the *last* thing you're working towards, there are other things as well

I wonder if there's a way to make it consistent for all bookmarked items...

**Review by malted — CHANGES_REQUESTED** [review]

**Pull request closed without merging**